spec/CHANGELOG.md

```diff
@@ -2,17 +2,17 @@
 
 ## Unreleased
 
-  - Added private messages. (#62)
+  - Added terms. (#62, #85)
 
-    Private messages start their identifiers with a single dash `-`.
+    Terms are entries which can only be referenced by other messages. Terms
+    cannot be retrieved by the `MessageContext` runtime.
 
-    Private messages cannot be retrieved by the `MessageContext` runtime.
-    Tools may introduce different checks for private and public messages.
+    Terms start their identifiers with a single dash `-`. Tools may introduce
```

Comment:
s/may/should/ ?

Reply:
I feel like this `may` makes sense. Fluent is static-analysis-friendly but isn't in itself a spec for how static analysis should be performed exactly.